File: cs50/week1/lecture-examples/ints.c
#include <cs50.h>
#include <stdio.h>

int main(){
    printf("x is ");
    int x = get_int();
    
    printf("y is ");
    int y = get_int();
    
    printf("%d + %d = %d\n", x, y, x + y);
    printf("%d - %d = %d\n", x, y, x - y);
    printf("%d * %d = %d\n", x, y, x * y);
    printf("%d / %d = %d\n", x, y, x / y);
    printf("%d %% %d = %d\n", x, y, x % y);
}

File: cs50/week8/lecture-examples/ints.py
import cs50

print("x is ", end = "")
x = cs50.get_int()

print("y is ", end = "")
y = cs50.get_int()

print("{} + {} = {}".format(x, y, x + y))
print("{} - {} = {}".format(x, y, x - y))
print("{} * {} = {}".format(x, y, x * y))
print("{} / {} = {}".format(x, y, x / y))
print("{} // {} = {}".format(x, y, x // y))
print("{} % {} = {}".format(x, y, x % y))